scripts/generate-app-config/generateAppConfig.js

```diff
@@ -85,8 +85,6 @@ const generateAppConfig = (siteConfig, production, verbose) => {
     navigationItems,
     extensions,
     indexPath: navConfig.navigation.index,
-    defaultLocale: appConfig.defaultLocale,
```

Comment:
Wait. do we want to drop this part?

Reply:
yes, it's no longer consumed anywhere. Since webpack will go directly to the site.config